### PR TITLE
[Script Lab] Add article

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
[Preview site](https://review.docs.microsoft.com/en-us/office/dev/add-ins/overview/explore-with-script-lab?branch=elizs-add-script-lab-article)

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[b06e76a](https://github.com/OfficeDev/office-js-docs-pr/pull/870/commits/b06e76a799e00d3b5737d87e0a2a94af5c82c407)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/screenshot-wide-youtube.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/images/screenshot-wide-youtube.png) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904032128594997-870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4DmjYWDtdK8rGkREyt2eYXjZsd0wDUtGZ%2FeCevVGtaY%3D&st=2019-04-03T21%3A25%3A25Z&se=2019-05-04T21%3A30%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/overview/explore-with-script-lab.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: Explore Office JavaScript API using Script Lab
+description: Learn about Script Lab and how you can use it to explore the Office JS API and prototype functionality.
+ms.topic: article
+ms.date: 04/03/2019
+localization_priority: Normal
+---
+
+# Explore Office JavaScript API using Script Lab
+
+As you begin or continue to develop your Office add-in, there are several ways to explore the Office JavaScript API (for example, quick starts and tutorials) but the [Script Lab add-in](https://aka.ms/getscriptlab), which is available free from the Office store, enables you to do so in a sort of sand-boxed environment. Script Lab is also a great way to prototype and verify functionality you want to include in your add-in.
```
Comment (AlexJerabek) [inline docs/overview/explore-with-script-lab.md]: This first sentence should be broken up into smaller sentences.

Comment (AlexJerabek) [inline docs/overview/explore-with-script-lab.md]: Consider leading with a statement about what Script Lab is.

File: docs/overview/explore-with-script-lab.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## What is Script Lab?
+
+Script Lab is a tool for anyone who wants to learn about developing Office Add-ins for Excel, Word, PowerPoint, and Project using Office JS API. It provides IntelliSense so you can see what's available. You can use existing samples in an Office host application to quickly try out something or use a sample as the base for your own. Another cool thing is that, through Script Lab, you might be able to try beta or preview functionality like [custom functions](/office/dev/add-ins/excel/custom-functions-overview).
```
Comment (AlexJerabek) [inline docs/overview/explore-with-script-lab.md]: Consider going a step further than saying it has Intellisense and point out it uses the Monaco framework VS Code uses (or just that it's like VS Code).

Comment (AlexJerabek) [inline docs/overview/explore-with-script-lab.md]: "you might be able to..." -> "you are able to"

File: docs/overview/explore-with-script-lab.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+> [!TIP]
+> In order to participate in beta or preview, you may have to sign up for the [Office Insider program](https://products.office.com/office-insider).
+
+Sounds good so far? Take a look at this one-minute teaser video to see Script Lab in action.
+
+[![Teaser video showing Script Lab being used in Excel, Word, and PowerPoint Online.](../images/screenshot-wide-youtube.png 'Script Lab teaser video')](https://aka.ms/scriptlabvideo)
+
+### Supported clients
+
+Script Lab is supported on the following clients for Excel, Word, PowerPoint, and Project where available.
```
Comment (AlexJerabek) [inline docs/overview/explore-with-script-lab.md]: Remove "where available"

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5c118e3](https://github.com/OfficeDev/office-js-docs-pr/pull/870/commits/5c118e37d43b61883c1f0d2c623cf2a8bf471ec7)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/screenshot-wide-youtube.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/images/screenshot-wide-youtube.png) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904191610391099-870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=N1ZYpEcXSoPcOtSZENGZZWx1jmwT2ku40UrAB%2BCyhrA%3D&st=2019-04-19T16%3A07%3A21Z&se=2019-05-20T16%3A12%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[cf28f56](https://github.com/OfficeDev/office-js-docs-pr/pull/870/commits/cf28f56bdd11d96d26c068c0374412a5845010d1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/screenshot-wide-youtube.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/images/screenshot-wide-youtube.png) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904191618129477-870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kbVedflS%2FdE%2Fg256cpzSIKWUoO%2BlwrRd5pzZ0hUBrKY%3D&st=2019-04-19T16%3A14%3A46Z&se=2019-05-20T16%3A19%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[df7e946](https://github.com/OfficeDev/office-js-docs-pr/pull/870/commits/df7e946586d1f0e274763c14d770ca1c7fb2a187)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/screenshot-wide-youtube.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/images/screenshot-wide-youtube.png) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904191638346331-870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=YVUEi%2FEudeRVojP9F%2BXngbyOBI0%2BFqaXnr5Et%2Fc%2BbEQ%3D&st=2019-04-19T16%3A35%3A08Z&se=2019-05-20T16%3A40%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[76f4f7e](https://github.com/OfficeDev/office-js-docs-pr/pull/870/commits/76f4f7e3b97b336d199eea536d4cca046ce33229)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/screenshot-wide-youtube.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/images/screenshot-wide-youtube.png) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C19%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904191642031353-870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=UTKk3ZBr14KXKds48rgkMnH%2FQVy1AtGQ0qX4k0h6ADw%3D&st=2019-04-19T16%3A38%3A25Z&se=2019-05-20T16%3A43%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/overview/explore-with-script-lab.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+## What is Script Lab?
+
+Script Lab is a tool for anyone who wants to learn about developing Office Add-ins using Office Javascript API in Excel, Word, and PowerPoint. It provides IntelliSense so you can see what's available and is built on the Monaco framework, the same framework as Visual Studio Code. Through Script Lab you can access a library of samples to quickly try out features or use a sample as the base for your own. You are also welcome to expand the sample library by adding snippets to the [office-js-snippets repo](https://github.com/OfficeDev/office-js-snippets#office-js-snippets). Another cool thing is you can try beta or preview functionality like [custom functions](/office/dev/add-ins/excel/custom-functions-overview).
+
+> [!TIP]
+> In order to participate in beta or preview, you may have to sign up for the [Office Insider program](https://products.office.com/office-insider).
+
+Sounds good so far? Take a look at this one-minute teaser video to see Script Lab in action.
+
+[![Teaser video showing Script Lab being used in Excel, Word, and PowerPoint Online.](../images/screenshot-wide-youtube.png 'Script Lab teaser video')](https://aka.ms/scriptlabvideo)
```
Comment (lindalu-MSFT) [inline docs/overview/explore-with-script-lab.md]:
```suggestion
[![Preiew video showing Script Lab running in Excel, Word, and PowerPoint Online.](../images/screenshot-wide-youtube.png 'Script Lab preview video')](https://aka.ms/scriptlabvideo)
```

File: docs/overview/explore-with-script-lab.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+## Supported clients
```
Comment (lindalu-MSFT) [inline docs/overview/explore-with-script-lab.md]:
```suggestion
## Script Lab supported clients
```

**Comment by lindalu-MSFT** [comment]
@ElizabethSamuel-MSFT Just a few minor suggestions, mainly "voice" edits and one change request. LMK if you have any questions!

File: docs/overview/explore-with-script-lab.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
+---
+title: Explore Office JavaScript API using Script Lab
+description: Use Script Lab to explore the Office JS API and to prototype functionality.
+ms.topic: article
+ms.date: 04/19/2019
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md]:
```suggestion
ms.date: 04/23/2019
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6e77e66](https://github.com/OfficeDev/office-js-docs-pr/pull/870/commits/6e77e66b5eed9dd6ac2ff6ff537cec99223d1418)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/screenshot-wide-youtube.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/images/screenshot-wide-youtube.png) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904231716494313-870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Sg1vSzXSmmUhtnEHw3B4%2BCERf%2BWS5L9O1tE%2FJbNh3zU%3D&st=2019-04-23T17%3A13%3A22Z&se=2019-05-24T17%3A18%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[0ee6280](https://github.com/OfficeDev/office-js-docs-pr/pull/870/commits/0ee6280995e10ed94fa358a9c51f36b062478246)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/screenshot-wide-youtube.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/images/screenshot-wide-youtube.png) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904231746288932-870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=97O32iTJPxHzrqsiM7nYKpVGHiEEPxV8hwm2XUY%2FA%2F0%3D&st=2019-04-23T17%3A42%3A58Z&se=2019-05-24T17%3A47%3A58Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/overview/explore-with-script-lab.md [hunk 7]
```diff
@@ -31,6 +31,10 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
 - Office 2013 or later for Windows
 - Office 2016 or later for Mac
 
+## Next steps
+
+When you're ready to create your Office Add-in, try out the [5-minute quick start](/office/dev/add-ins/#5-minute-quick-starts) for your preferred Office application.
```
Comment (lindalu-MSFT) [inline docs/overview/explore-with-script-lab.md]: When you're ready to create your own Office Add-in, see the [5-minute quick start](/office/dev/add-ins/#5-minute-quick-starts) for your preferred Office application.

File: docs/overview/explore-with-script-lab.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+- Office 365 for Windows
+- Office 365 for Mac
+- Office Online
+- Office 2013 or later for Windows
+- Office 2016 or later for Mac
+
+## Next steps
+
+When you're ready to create your Office Add-in, try out the [5-minute quick start](/office/dev/add-ins/#5-minute-quick-starts) for your preferred Office application.
```
Comment (ElizabethSamuel-MSFT) [inline docs/overview/explore-with-script-lab.md]:
```suggestion
When you're ready to create your Office Add-in, see the [5-minute quick start](/office/dev/add-ins/#5-minute-quick-starts) for your preferred Office application.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[8231eee](https://github.com/OfficeDev/office-js-docs-pr/pull/870/commits/8231eee8222cc10d1e9a1ac38258b3557a69dd48)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/screenshot-wide-youtube.png](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/images/screenshot-wide-youtube.png) | :white_check_mark:Succeeded | |
[docs/overview/explore-with-script-lab.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/overview/explore-with-script-lab.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-add-script-lab-article/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904232306251905-870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sTTi4TIYmq5uqgPKRdA88OItbhj1TkVL84oC9cXT6jI%3D&st=2019-04-23T23%3A02%3A59Z&se=2019-05-24T23%3A07%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**